python/pysrc/kaskada/_timestream.py

```diff
@@ -880,6 +893,84 @@ def record(self, fields: Callable[[Timestream], Mapping[str, Arg]]) -> Timestrea
         """
         return record(fields(self))
 
+    def round(self) -> Timestream:
+        """Return a Timestream with all values rounded to the nearest integer.
+
+        Returns:
+            A Timestream of the same type as `self`. The result contains `null`
+              if the value was `null` at that position. Otherwise, it contains
+              the result of rounding the value to the nearest integer.
+
+        Notes:
+            This method may be applied to any numeric type. For anything other
+              than `float32` and `float64` it has no affect since the values
+              are already integers.
+
+        See Also:
+            `ceil` and `floor`
```

Comment:
Double check the rendering (`nox -s docs`) to make sure both of these are linked. I'm not sure. You may need to do:

```
See Also:
  ceil
  floor
```

or possibly comma separated? Not sure what Sphinx expects here.

Reply:
needs to be like this:
```
        See Also:
            :func:`ceil`
            :func:`floor`
```